contrib/sly-package-fu.el

```diff
@@ -317,4 +332,126 @@ symbol in the Lisp image if possible."
 
 (defalias 'sly-export-structure 'sly-export-class)
 
+;; 
+;; Dealing with import-from
+;;
+;; MELPA version: (buffer-substring-no-properties point1 pount2)
+;; ~/.emacs.d/elpa/sly-20171111.1604/contrib/sly-package-fu.el
+
+(defun sly-search-import-from (package)
+  ;; Suppose, we are in the defpackage sexp
+  (let* ((normalized-package (sly-package-normalize-name package))
+         (regexp (format "(:import-from[ \t']*\\(:\\|#:\\)?%s"
+                         (regexp-quote (regexp-quote normalized-package))))
+         (search-result (re-search-forward regexp nil t)))
+    (message "Normalized: %s, regex: %s" normalized-package
+             regexp)
+    (when search-result
+      ;; import-from clause was found
+      t)))
+
+
+(defun sly-package-write-imported (symbol)
```

Comment:
Please delete this definition if you don't use it anywhere.

Reply:
Good catch. Removed.